systemd/docker.service

```diff
@@ -9,9 +9,8 @@ Wants=network-online.target
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd
+ExecStart=/usr/bin/dockerd -H unix://
```

Comment:
I'm out of the loop with @tiborvass's proposed engine change - is this the "final" state we need to get to, or an intermediate step?